### DISCUSSION TITLE
Left Handed Mode (Or the ability to change keybinds for z & x)

### DISCUSSION THREAD

**Discussion by RealCrayfish** [ppy/osu#16215] [Ideas] [answered]
I don't know how many this will apply to but for left handed keyboard players maybe give the option to switch `z` and `x` to something like `n` and `m`

**Comment by bdach** [comment, marked as answer]
like this you mean?

![2021-12-22-222922_658x523_scrot](https://user-images.githubusercontent.com/20418176/147157376-2a75a8ae-4e36-4b05-9f34-73cb6ada3648.png)

this is already in the game (settings -> search for "bindings" -> configure -> go to osu! section)
